aiida_firecrest/transport.py

```diff
@@ -135,16 +282,29 @@ def __init__(
         url: str,
         token_uri: str,
         client_id: str,
-        client_secret: str | Path,
+        client_secret: str,  # | Path,
+        # unfortunately we cannot store client_secret as a Path, because it is not JSON serializable
         client_machine: str,
-        small_file_size_mb: float = 5.0,
-        file_transfer_poll_interval: float = 0.1,
+        small_file_size_mb: float,
+        temp_directory: str,
+        # configured: bool = True,
         # note, machine is provided by default,
         # for the hostname, but we don't use that
         # TODO ideally hostname would not be necessary on a computer
         **kwargs: Any,
     ):
-        """Construct a FirecREST transport."""
+        """Construct a FirecREST transport object.
+
+        :param url: URL to the FirecREST server
+        :param token_uri: URI for retrieving FirecREST authentication tokens
+        :param client_id: FirecREST client ID
+        :param client_secret: FirecREST client secret or Absolute path to an existing FirecREST Secret Key
```

Comment:
```suggestion
        :param client_secret: FirecREST client secret to an existing FirecREST Secret Key
```
please update if you choose not to support Path

Reply:
Thanks for noticing this!
In fact this should be absolute path but only in `str` format not an instance of `Path`, how to say this?